packages/pymedphys_sphinxtheme/src/sass/_theme_badge.sass

```diff
@@ -92,8 +92,8 @@
 
 
 .rst-versions .rst-current-version
-  background-color: #eaeaea
-  color: #55a5d9
+  background-color: #dddddd
+  color: #0077CC
```

Comment:
What does this change?

Reply:
I'm okay with this version colouring, but was leaving this change in there intentional?

Reply:
It was intentional for the colour change, but I meant to revert it for the border option. I'll revert it :)